dependencies64/ffmpeg/ffmpeg-build-linux/build-ubuntu.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+#compile ffmpeg successful on ubuntu 14.04.3 & 16.04.1 amd64 using the CasparCG build requirments described here:
+#https://raw.githubusercontent.com/casparcg/Server/2.1.0/BUILDING
 
-sudo apt-get install build-essential curl tar pkg-config
+sudo apt-get -y update
+sudo apt-get -y upgrade
+sudo apt-get -y install build-essential curl tar pkg-config python2.7-dev zlib1g-dev autoconf libtool subversion libfontconfig1-dev ant default-jre default-jdk frei0r-plugins-dev libv4l-dev
 
```

Comment:
Only to "build" ffmpeg for casparcg on top of the build requirements of casparcg it self see BUILDING (build-essential cmake libxrandr-dev libjpeg-dev libsndfile1-dev libudev-dev libglu1-mesa-dev). This has been tested on to clean installs of ubuntu 14.04.3 and 16.04.1 this weekend. 

To "run" CasparCG server after building it requires libgcrypt on ubuntu 16.04.1 which is mentioned by @baltedewit #533 

To summarize the current CasparCG BUILDING requirements/packages and these FFmpeg related requirements/packages are enough to build CasparCG and ffmpeg on ubuntu 14.04.3-higher 16.04.1.